.circleci/config.yml

```diff
@@ -118,10 +118,16 @@ jobs:
         type: enum
         enum: [ 'x64', 'arm64','armv7l' ]
     docker:
-      - image: cimg/base:stable
+      - image: ubuntudesktop/gnome-3-38-2004
```

Comment:
What determines what version we use here? I found [this](https://hub.docker.com/layers/ubuntudesktop/gnome-3-38-2004/latest/images/sha256-f22e93b35de184d01b46c99e6c3975ea7aa0391d7ffde61404809029cb9a919b?context=explore) which looks like it's not been updated in ~2 years - could that present a problem in future?